api/src/main/java/io/smallrye/stork/integration/StorkInfrastructure.java

```diff
@@ -0,0 +1,7 @@
+package io.smallrye.stork.integration;
+
+import java.util.function.Supplier;
+
+public interface StorkInfrastructure {
+    <T> T get(Class<T> utilityClass, Supplier<T> defaultSupplier);
```

Comment:
Javadoc missing

It should specify what should happen if the default supplier throws a runtime exception

Reply:
I added the javadoc. What happens with exceptions thrown by supplier depends on an implementation...